src/main/scala/uk/gov/nationalarchives/DisasterRecoveryItem.scala

```diff
@@ -0,0 +1,22 @@
+package uk.gov.nationalarchives
+
+import cats.effect.IO
+import fs2.io.file._
+
+import java.util.UUID
+import scala.xml.Elem
+
+sealed trait DisasterRecoveryItem {
```

Comment:
We're using "Items" and "Objects" synonymously (mainly using "Items" up until we call the `writeObjects`/`updateObjects` methods); should we stick to one?

Reply:
It's a fair point, they're called objects in Preservica and objects in OCFL so I've changed all the "items" to "objects"